corehq/apps/sms/models.py

```diff
@@ -427,3 +443,75 @@ def case_changed_receiver(sender, case, **kwargs):
 
 case_post_save.connect(case_changed_receiver, CommCareCase)
 
+
+class PhoneNumber(models.Model):
+    """
+    Represents a single phone number. This is not intended to be a
+    comprehensive list of phone numbers in the system (yet). For
+    now, it's only used to prevent sending SMS/IVR to phone numbers who
+    have opted out.
+    """
```

Comment:
@czue yeah, I think longer term we can move the whole phone registry to this table, but for now just wanted to get it out there to use it for this effort